patchAPI/example/mutatee/main.c

```diff
@@ -0,0 +1,34 @@
+
+#include <stdio.h>
+#include <unistd.h>
+
+
+static int status = -1;
+
+void foo0(void){
+  status = 0;
+}
+
+void foo1(void) {
+  status = 1;
+}
+
+void foo2(void) {
+  status = 2;
+}
+
+void message(const char* msg){
+  printf("%s\n", msg);
+}
+
+int main(int argc, const char *argv[])
+{
+  if (argc > 2){
+    foo0();
+  } else {
+    foo1();
+    foo2();
+  }
+  message("Done\n");
+  return status;
```

Comment:
Returning non-zero indicates failure to the shell. Is that relevant here?

Reply:
I fixed this, plus added a remark about what the rewritten program depends on
(i.e. the dyninst runtime library).